main/include/LockManager.h

```diff
@@ -16,15 +16,15 @@
  *    limitations under the License.
  */
 
-#ifndef LOCK_WIDGET_H
-#define LOCK_WIDGET_H
+#ifndef LOCK_MANAGER_H
+#define LOCK_MANAGER_H
 
 #include <stdint.h>
 #include <stdbool.h>
 
 #include "AppEvent.h"
 
-class LockWidget
+class LockManager
```

Comment:
Just a thought: I kind of wish this was called BoltLockManager, as I think that matches the trait name better.

Reply:
Granted. You now have 29 wishes left.